test/drawers/drawerTests.ts

```diff
@@ -22,17 +21,17 @@ class MockAnimator implements Plottable.Animator {
 }
 
 function createMockDrawer(dataset: Plottable.Dataset) {
-  var drawer = new Plottable.Drawer(dataset);
+  const drawer = new Plottable.Drawer(dataset);
```

Comment:
In regards to how the switching idea is key.  Let's say I have the below code
```typescript
let tester = foo;
tester([1, 3, 4]);
tester = bar;
tester([1, 3, 4]);
```

The semantics of the above would essentially read:
I have a variable tester that can be switched out to other testers to operate on some array.  Another way to word it is that essentially the value of `tester` changes drastically between lines.

However, what we can do is reduce the switching with `const`
```typescript
const fooTester = foo;
fooTester([1, 3, 4]);
const barTester = bar;
barTester([1, 3, 4]);
```

The semantics become a lot more clear, in my opinion.